subprojects/packagefiles/sigdb/meson.build

```diff
@@ -0,0 +1,3 @@
+project('sigdb', version: 'main')
+
+install_subdir('.', install_dir : get_option('install_dir'))
```

Comment:
What about adding this entire meson build system to the sigdb repo?

Reply:
yes, this can be done.

Reply:
moved to sigdb